File: BusinessLayer/Model/EmployeeModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace BusinessLayer
{
    public class EmployeeModel
    {
        #region " Public Properties "

        public int EmpId { get; set; }
        public string EmpName { get; set; }
        public string DateOfBirth { get; set; }
        public string DateOfJoin { get; set; }
        public int Salary { get; set; }
        public string Gender { get; set; }
        public string Operation { get; set; }

        #endregion
    }
}

File: CRUDwithAngularAndWebAPI/Angular/src/app/employee.model.js
System.register([], function (exports_1, context_1) {
  "use strict";
  var __moduleName = context_1 && context_1.id;
  var Employee;
  return {
    setters: [],
    execute: function () {
      Employee = (function () {
        debugger;
        function Employee(EmpId, EmpName, Gender, DateOfBirth, DateOfJoin, Salary, Operation) {
          this.EmpId = EmpId;
          this.EmpName = EmpName;
          this.Gender = Gender;
          this.DateOfBirth = DateOfBirth;
          this.DateOfJoin = DateOfJoin;
          this.Salary = Salary;
          this.Operation = Operation;
        }
        return Employee;
      }());
      exports_1("Employee", Employee);
    }
  };
});
